frontend/app/src/components/hexagonalTile/HexagonalTile.tsx

```diff
@@ -0,0 +1,47 @@
+import "./HexagonalTile.css";
+
+export interface HexagonalTileProps {
+	fill: string;
+	isDimmed: boolean;
+	stroke: string;
+	strokeWidth: string;
+	identifier: string;
+	textContent: string;
+}
+
+function HexagonalTile(
+	props: HexagonalTileProps = {
+		fill: "",
+		isDimmed: false,
+		strokeWidth: "1",
+		stroke: "gray",
+		identifier: "",
+		textContent: "",
+	},
```

Comment:
since we destructe the props everywhere else too, for consistency reasons i would do the same here. My suggestion would be this:
	{
		fill= "",
		isDimmed= false,
		strokeWidth= "1",
		stroke= "gray",
		identifier= "",
		textContent= "",
	}:Readonly\<HexagonalTileProps\>
+ replace props.{attributeName} with {attributeName}

Reply:
Implemented suggestion of @Straczi, agree that it seems nicer